frontend/src/bundles/common/components/router-provider/router-provider.tsx

```diff
@@ -31,7 +31,7 @@ const RouterProvider: React.FC<Properties> = ({ routes }) => {
                     ...(children?.length && {
                         children: mapRoutes(
                             children,
-                            isAlreadyProtected || shouldProtect,
+                            isAlreadyProtected ?? shouldProtect,
```

Comment:
invalid, it will not work with nullish coalescing since first operand is boolean